Cargo.toml

```
@@ -1,11 +1,11 @@
 [package]
 name = "tokio-uring"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["Carl Lerche <me@carllerche.com>"]
```

Seems to me that we should update this now that the project is in the Tokio org.
```suggestion
authors = ["Tokio Contributors <team@tokio.rs>"]
```